blocks/product-details/product-details.js

```diff
@@ -128,8 +132,9 @@ export default async function decorate(block) {
     pdpRendered.render(ProductOptions, { hideSelectedValue: false })($options),
 
     // Configuration  Quantity
-    pdpRendered.render(ProductQuantity, {})($quantity),
-
+    pdpRendered.render(ProductQuantity, {
+      defaultQuantity: isEditMode ? urlQuantity : 1,
+    })($quantity),
```

Comment:
I couldn't see the quantity from URL being applied to the quantity incrementer in the PDP page using the test URL of this PR, but since this was already implemented in PR #419, we could remove this. 
```suggestion
```